packages/api/src/routes/medical/schemas/request-metadata.ts

```diff
@@ -0,0 +1,5 @@
+import { z } from "zod";
+
+export const cxRequestMetadataSchema = z.object({
+  meta: z.record(z.string().min(1).max(40), z.string().max(40).optional()).optional(),
```

Comment:
optional metadata

Reply:
Do we also want the value to be limited to 40 chars? Stripe uses 500 and I suggested 256, just trying to understand the thought process.